code/modules/power/cable.dm

```diff
@@ -671,7 +671,7 @@ var/global/list/datum/stack_recipe/cable_coil_recipes = list(
 
 	if(C.shock(user, 50))
 		if(prob(50)) //fail
-			new /obj/item/stack/cable_coil(get_turf(C), 1, C.color)
+			new /obj/item/stack/cable_coil(get_turf(C), 1, paramcolor = C.color)

```

Comment:
Is adding `paramcolor = ` necessary here?

Reply:
Necessary, else I will have to fill in everything by positional arguments to prevent a runtime caused by passing in a color to a boolean/number field (The merge field)